### ISSUE TITLE
Viruses don't cure

### ISSUE THREAD

**Issue by LanCartwright** [tgstation/-tg-station#13619]
Retrovirus doesn't cure with mutadone.

My branch has this fixed, but I don't know when or where.

This bug also affects other special viruses such as brainrot so you want to test all of them.

**Comment by Anonus** [comment]
Not actually a bug, retrovirus is cured with either mutadone or lying down until you get a message.

**Comment by LanCartwright** [comment]
not true, I injected 4 people with more than 10-15u of mutadone each. None of them were cured.

**Comment by Knigga** [comment]
At one point I had ingested at least 60u of mutadone and it didn't even drop a stage

**Comment by Anonus** [comment]
OR, it can't be cured with both, it's one or the other. The med analyzer message says this, though it doesn't note it's an exclusive or.

**Comment by Anonus** [comment]
[The code, for reference](https://github.com/tgstation/-tg-station/blob/master/code/datums/diseases/retrovirus.dm)
The med analyzer message is wrong, it actually still refers to ryetalyn, but it's definitely not a bug.

**Comment by LanCartwright** [comment]
If it's ryetlyn it's bugged because you can't get it. If it's mutadone it's bugged because it doesn't cure.

**Comment by RemieRichards** [comment]
it's mutadone OR it's lying.
it's not both
the ryetlyn thing is what it was in the past

tl;dr:

## past:
rytelyn OR lying

## now:
mutadone OR lying

**Comment by optimumtact** [comment]
the confusion is because when we say or we mean that the cure type is literally chosen from either lying or mutadone at virus creation, so you have to figure out the cure type and then use that. at least that was the way it was explained to me
